src/Codeception/Lib/Driver/PostgreSql.php

```diff
@@ -42,12 +48,12 @@ public function load($sql)
                 $this->searchPath = $match[0];
             }
 
-            $query .= "\n" . rtrim($sqlLine);
+            $query .= "\n".rtrim($sqlLine);
```

Comment:
Please put spaces around concatenation operator. This isn't specified by PSR-2, but it is common convention followed by many coding standards and elsewhere in Codeception.